src/app/templates/run-report-page.ftl

```diff
@@ -1,16 +1,20 @@
 <#-- @ftlvariable name="runReportMetadataJson" type="String" -->
 <#-- @ftlvariable name="gitBranchesJson" type="String" -->
+<#-- @ftlvariable name="initialReportName" type="String" -->
 <@layoutwide>
     <#macro styles>
         <link rel="stylesheet" href="${appUrl}/css/report-page.min.css"/>
     </#macro>
-    <div id="runReportTabsVueApp">
-        <run-report-tabs :metadata="runReportMetadata" :initial-git-branches="gitBranches"></run-report-tabs>
-    </div>
+        <div id="runReportTabsVueApp">
+            <run-report-tabs :metadata="runReportMetadata"
+                             :initial-git-branches="gitBranches"
+                             :report-name="initialReportName"></run-report-tabs>
```

Comment:
Were you not going to rename the props on the components too? I think that's where it's going to provide most clarification.